app/controllers/mixins/ems_common_angular.rb

```diff
@@ -478,6 +478,9 @@ def get_hostname_from_routes(ems, endpoint_hash, token)
     rescue KubeException, OpenSSL::SSL::SSLError => e
```

Comment:
If you're keeping this separate to log "error" vs "unexpected error", consider upgrading to `OpenSSL::OpenSSLError` (see #1120 explanation).